x-pack/plugins/upgrade_assistant/public/application/components/es_deprecations/deprecation_types/default/table_row.tsx

```diff
@@ -44,6 +44,7 @@ export const DefaultTableRow: React.FunctionComponent<Props> = ({ rowFieldNames,
         },
         flyoutProps: {
           onClose: closeFlyout,
+          className: 'eui-textBreakWord',
```

Comment:
Instead of applying the class to all the strings inside the flyout bodies i just decided to apply it to the wrapper to avoid making lots of tiny changes, unsure if we have any rules about this but LMK otherwise and I can change it

Reply:
I had added it to the default flyout in https://github.com/elastic/kibana/pull/110843. Is that still necessary or can it be removed there?

Reply:
I believe we will still need it since some of the other flyouts can have an index name in the title or body that can make it overflow